Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -920,6 +931,25 @@ def addVertexFitting(
 
     customLogLevel = acts.examples.defaultLogging(s, logLevel)
 
+    selectedTrackParameters = "trackparameters"
+    trackIndices = "outputTrackIndices"
+
+    if trackSelectorRanges is not None:
+        trackSelector = acts.examples.TrackSelector(
+            level=customLogLevel(),
+            inputTrackParameters=trackParameters,
+            outputTrackParameters=selectedTrackParameters,
+            outputTrackIndices=trackIndices,
+            **acts.examples.defaultKWArgs(
+                removeNeutral=trackSelectorRanges.removeNeutral,
+                absEtaMax=trackSelectorRanges.absEta[1],
+                ptMin=trackSelectorRanges.pt[0],
```

Comment:
```suggestion
                absEtaMin=trackSelectorRanges.absEta[0],
                absEtaMax=trackSelectorRanges.absEta[1],
                ptMin=trackSelectorRanges.pt[0],
                ptMax=trackSelectorRanges.pt[1],
```
and then add all the other selectors. Sorry, I didn't write them all out, but it's not so easy in the comments. If you are having trouble with this, I can make a commit of my own.